docs/internals/permissions/project-permissions.mdx

```diff
@@ -0,0 +1,279 @@
+| `create` | Create new tags for organizing resources |
+| `edit`   | Modify existing tags                     |
+| `delete` | Remove tags from the project             |
+
+#### Subject: `workspace`
+| Action   | Description               |
+| -------- | ------------------------- |
+| `edit`   | Modify workspace settings |
+| `delete` | Delete the workspace      |
+
+#### Subject: `ip-allowlist`
+| Action   | Description                                  |
+| -------- | -------------------------------------------- |
+| `read`   | View IP allowlists                           |
+| `create` | Add new IP addresses or ranges to allowlists |
+| `edit`   | Modify existing IP allowlist entries         |
+| `delete` | Remove IP addresses from allowlists          |
+
+#### Subject: `audit-logs`
+| Action | Description                                             |
+| ------ | ------------------------------------------------------- |
+| `read` | View audit logs of actions performed within the project |
+
+#### Subject: `integrations`
+| Action   | Description                      |
+| -------- | -------------------------------- |
+| `read`   | View configured integrations     |
+| `create` | Add new third-party integrations |
+| `edit`   | Modify integration settings      |
+| `delete` | Remove integrations              |
+
+#### Subject: `webhooks`
+| Action   | Description                          |
+| -------- | ------------------------------------ |
+| `read`   | View webhook configurations          |
+| `create` | Add new webhooks                     |
+| `edit`   | Modify webhook endpoints or triggers |
+| `delete` | Remove webhooks                      |
+
+#### Subject: `service-tokens`
+| Action   | Description                              |
+| -------- | ---------------------------------------- |
+| `read`   | View service tokens                      |
+| `create` | Create new service tokens for API access |
+| `edit`   | Modify token properties                  |
+| `delete` | Revoke or remove service tokens          |
+
+### Secrets Management
+
+#### Subject: `secrets`
+Supports conditions and permission inversion
+| Action   | Description                     |
+| -------- | ------------------------------- |
+| `read`   | View secrets and their values   |
+| `create` | Add new secrets to the project  |
+| `edit`   | Modify existing secret values   |
+| `delete` | Remove secrets from the project |
+
+#### Subject: `secret-folders`
+Supports conditions and permission inversion
+| Action   | Description              |
+| -------- | ------------------------ |
+| `read`   | View secret folders      |
+| `create` | Create new folders       |
+| `edit`   | Modify folder properties |
+| `delete` | Remove secret folders    |
+
+#### Subject: `secret-imports`
+Supports conditions and permission inversion
+| Action   | Description           |
+| -------- | --------------------- |
+| `read`   | View secret imports   |
```

Comment:
We need some kinda of workflow to update this periodically or whenever we add new permission

Reply:
Created linear for it https://linear.app/infisical/issue/ENG-2505/create-workflow-to-keep-permissions-docs-automatically-updated-with

Reply:
agreed ^